### ISSUE TITLE
Intention ExtractStringKey is not writing to arb files. 

### ISSUE THREAD

**Issue by Schrall** [localizely/flutter-intl-intellij#74]
When i extract a String with ExtractStringKey to generate an ARB entry. 

From:
`return 'Please enter some text';`
to 
`return S.of(context).pleaseEnterSomeText; `

The translation is missing in all the selected Files:

{
    "Login": "Login"
}

Intellij 2021.2.2 (Ultimate) 
Flutter Intl 1.15.0-2019.2
macOS Big Sur 11.6

Read and Write Permission for Everybody on the whole Projekt. 
<img width="553" alt="Bildschirmfoto 2021-10-16 um 12 58 24" src="https://user-images.githubusercontent.com/7008429/137584655-580d7748-231c-4a31-8627-b662269af714.png">
<img width="452" alt="Bildschirmfoto 2021-10-16 um 12 57 39" src="https://user-images.githubusercontent.com/7008429/137584656-5346af13-0069-4978-9d54-25cf3cdf4821.png">
<img width="399" alt="Bildschirmfoto 2021-10-16 um 12 57 49" src="https://user-images.githubusercontent.com/7008429/137584657-f48fd81b-278d-4248-8f62-bba65edb0c3d.png">

**Comment by lzoran** [comment]
Hi @Schrall,

Thanks for reporting this!

I've tried to reproduce this issue, but without success. 😕 

I noticed that you are using the older version of the Flutter Intl plugin (`1.15.0-2019.2`) with the newer version of the IDE (`2021.2.2`).
Could you try to upgrade your Flutter Intl plugin to the `1.15.0-2020.3` version and check if the reported issue still occurs?

**Comment by lzoran** [comment]
I'm closing this issue now.
In case the same problem occurs after the plugin update, feel free to open it again.

**Comment by KhanhPham0102** [comment]
Hi @lzoran,
I still get this error in the new version.

Intellij 2021.3 (Ultimate)
Flutter Intl 1.17.0-2020.3
Win 11 Home.

Does it need any additional settings to use?

**Comment by lzoran** [comment]
Hi @KhanhPham0102,

I've tested string key extraction with your settings on my machine, and it seems to work well. 😕

![extact-string-key](https://user-images.githubusercontent.com/15170421/148039751-4b1b8526-1a36-4093-a7f2-15523921212f.gif)

A minimal reproducible example is welcome.

> Does it need any additional settings to use?

No, you just need to have Flutter Intl plugin enabled for your Flutter project.

**Comment by KhanhPham0102** [comment]
Hi @lzoran ,

This is the problem I recorded. 

https://user-images.githubusercontent.com/35913261/148421338-1e799490-c567-4db6-8b40-b60e37b21806.mp4

It is not added to the .arb file so reloading the .arb file automatically is not done. 

My config in pubspec.yaml :

![image](https://user-images.githubusercontent.com/35913261/148422226-32601446-9588-44b9-91a6-7bf638b29d7a.png)

**Comment by lzoran** [comment]
Hi @KhanhPham0102,

Thanks for provided materials!

I've tried again to reproduce this issue, but without success. Everything works well on my machine. 😞

I noticed few interesting things on your screen recording:

- ARB files does not have their icons under project tree. Maybe that is custom setting in your IDE or plugin responsible for that does not work well (I assume that you have enabled `Dart (JetBrains)`, `Flutter (flutter.dev)` and `Flutter Intl (Localizely)` plugins).

- Extraction does not trigger generation of localization code. Once you confirm the extraction of a string key, the plugin should update appropriate ARB files and trigger generation.

Not sure what might cause this issue. 😕 
Maybe re-installing `Flutter Intl` plugin and IDE restart might help.

Also, does the same issue appears on other Flutter projects? Maybe you can try to reproduce this issue on [this sample project](https://github.com/localizely/flutter-intl-plugin-sample-app).

**Comment by w568w** [comment]
Same to me on `Android Studio Arctic Fox | 2020.3.1 Patch 4`, `Windows 11` & `Flutter Intl 1.17.0-2020.3`. 

No arb file updated and no generation triggered. Tried on different OS (Ubuntu, Debian, Windows 10…) with clean installed IDE and the sample project, but no luck. 🤕

**Comment by jakub-trzebiatowski** [comment]
For me, "Extract to ARB" doesn't work even for the sample project.

**Comment by jakub-trzebiatowski** [comment]
@lzoran Inspired by your observation about the ARB files icons, I checked that out, and I also did't have any fancy icons on ARB files. I checked the File Types settings:

![image](https://user-images.githubusercontent.com/2590174/199477926-28f73307-e27c-425d-9ee9-30e98688b221.png)

The ARB file type didn't have any file name pattern registered. Registering `*.arb` fixed the issue.

![image](https://user-images.githubusercontent.com/2590174/199478179-b3ebaf48-d75f-4ba6-b40a-e37582b43e0b.png)
